config/cdash.php

```diff
@@ -64,7 +67,12 @@
     'show_last_submission' => env('SHOW_LAST_SUBMISSION', true),
     'slow_page_time' => env('SLOW_PAGE_TIME', 10),
     'token_duration' => env('TOKEN_DURATION', 15811200),
-    'validate_xml_submissions' => env('VALIDATE_XML_SUBMISSIONS', false),
+    'validate_submissions' => (match (Str::upper((string) env('VALIDATE_SUBMISSIONS'))) {
+        'SILENT' => SubmissionValidationType::SILENT,
+        'WARN' => SubmissionValidationType::WARN,
+        'REJECT' => SubmissionValidationType::REJECT,
+        default => SubmissionValidationType::SILENT,
+    }),
```

Comment:
I believe these parentheses are unnecessary:

```suggestion
    'validate_submissions' => match (Str::upper((string) env('VALIDATE_SUBMISSIONS'))) {
        'SILENT' => SubmissionValidationType::SILENT,
        'WARN' => SubmissionValidationType::WARN,
        'REJECT' => SubmissionValidationType::REJECT,
        default => SubmissionValidationType::SILENT,
    },
```

Reply:
Done!